proposals/community-strategy/social-media.md

```diff
@@ -0,0 +1,41 @@
+# Social media
+
+## Youtube
+
+[Youtube](https://www.youtube.com/) is an important resource for development. 
+Especially when it comes to modern web frameworks, such as React, VueJS or Angular. Some development Youtube channels 
+reach some millions of developers.
+
+### Some Youtube channels in the relevant area
+
+#### Embedded
+
+- [Andreas Spiess](https://www.youtube.com/channel/UCu7_D0o48KbfhpEohoP7YSQ)
+- [GreatScott!](https://www.youtube.com/user/greatscottlab)
+- [Make:](https://www.youtube.com/c/MAKE/featured)
+
+#### Web development
+
+- [Traversy Media](https://www.youtube.com/c/TraversyMedia/featured)
+- [freeCodeCamp.org](https://www.youtube.com/c/Freecodecamp/videos)
+- [Dev Ed](https://www.youtube.com/c/DevEd/videos)
+
+A dedicated Youtube channel with high quality videos about Web of Things would help new users to get into the topic.
+Cooperative content with big Youtube development Youtube Channel will also help to reach more users.
+
+## Stackoverflow
+
+[Stackoverflow](https://stackoverflow.com/) is the biggest question and answer site for development related questions. 
+To be present on Stackoverflow is a necessity in order to not miss questions by developer using the standard.
+
```

Comment:
I agree; we should reach a consensus about a possible tag to be used there. What about a simple `webofthings` for general questions and `node-wot` for js/ts implementations?.

Reply:
Btw I can create tags on stackoverflow.

Reply:
> Btw I can create tags on stackoverflow.

Awesome! Could you please create the following list for us?

`web-of-things`, `node-wot`, `thing-description`, `w3c-wot`, and `wot`.

I used dashes as separators. it seems to be the convention there. Please let us know when you are done so that we can subscribe to those new tags.  Thank you!

Reply:
I don't think they approve multiple tags for the same thing. So, I think web-of-things is probably the best tag for the purpose. So: `web-of-things`, `node-wot` and `thing-description`

Reply:
[web-of-things](https://stackoverflow.com/questions/tagged/web-of-things) and [thing-description](https://stackoverflow.com/questions/tagged/thing-description) are available on stackoverflow.